usaspending_api/etl/transaction_loaders/field_mappings_fpds.py

```diff
@@ -508,14 +511,15 @@
     "recipient_flag": lambda broker: True,
     "create_date": current_datetime,  # Data loader won't add this value if it's an update
     "update_date": current_datetime,
+    "zip5": legal_entity_zip5,
+    "state_code": legal_entity_state_code,
+    "state_description": legal_entity_state_description,
 }
 
 # broker column name -> usaspending column name
 place_of_performance_nonboolean_columns = {
     "place_of_perform_country_c": "location_country_code",
     "place_of_perform_country_n": "country_name",
-    "place_of_performance_state": "state_code",
-    "place_of_perfor_state_desc": "state_description",
```

Comment:
I can't comment on line 528:
``` "place_of_performance_zip5": "zip5",```
Should that be removed from `place_of_performance_nonboolean_columns`?

Reply:
it should